core/src/main/java/org/apache/iceberg/FileMetadata.java

```diff
@@ -57,6 +59,8 @@ public static class Builder {
     private ByteBuffer keyMetadata = null;
     private Integer sortOrderId = null;
 
+    private List<Long> splitOffsets = null;
+
```

Comment:
Just a nit, could we move the field just below `sortOrderId` and avoid the unnecessary new line